.travis.yml

```diff
@@ -19,11 +10,30 @@ addons:
     bucket: sequence-viewer-artifacts
     paths:
     - reports
-before_install:
-- export JAVA_HOME=/usr/lib/jvm/java-8-oracle
-- export PATH=$JAVA_HOME/bin:$PATH
-script:
-- concurrently -p name -n lint:js,lint:wc,polymer:test "npm run lint:js" "npm run lint:wc" "polymer test --skip-plugin local" && npm run test:galen:sauce
-env:
-  global:
-    SAUCE_USERNAME: Desire2Learn
```

Comment:
is this in the secure environment variable now?

Reply:
Yes